chia/server/server.py

```diff
@@ -157,15 +164,47 @@ def __init__(
         if self._local_type is NodeType.INTRODUCER:
             self.introducer_peers = IntroducerPeers()
 
-        if self._local_type is not NodeType.INTRODUCER:
-            self._private_cert_path, self._private_key_path = private_ssl_paths(root_path, config)
-        if self._local_type is not NodeType.HARVESTER:
-            self.p2p_crt_path, self.p2p_key_path = public_ssl_paths(root_path, config)
+        ca_private_crt_path, ca_private_key_path = private_ca_crt_key
+        chia_ca_crt_path, chia_ca_key_path = chia_ca_crt_key
+
+        private_cert_path, private_key_path = None, None
+        public_cert_path, public_key_path = None, None
```

Comment:
I personally would rather see a line per for this.  I don't think the build-a-tuple-then-unpack-it does much good here.

```suggestion
        private_cert_path = None
        private_key_path = None
        public_cert_path = None
        public_key_path = None
```

Or even add the hints here.